api/r/cellxgene.census/R/open.R

```diff
@@ -1,47 +1,63 @@
-DEFAULT_TILEDB_CONFIGURATION <- c(
-  "py.init_buffer_bytes" = paste(1 * 1024**3),
-  "soma.init_buffer_bytes" = paste(1 * 1024**3)
-)
-
 #' Open the Cell Census
 #'
 #' @param census_version The version of the Census, e.g., "latest".
 #' @param uri The URI containing the Census SOMA objects. If specified, takes
 #'            precedence over `census_version`.
 #' @param tiledbsoma_ctx A custom `tiledbsoma::SOMATileDBContext`
 #'
-#' @return Top-level `tiledbsoma::SOMACollection` object
+#' @return Top-level `tiledbsoma::SOMACollection` object. After use, the census
+#'         should be closed to release memory and other resources, usually with
+#'         `on.exit(census$close(), add = TRUE)`. Closing the top-level census
+#'         will also close all SOMA objects accessed through it.
 #' @importFrom tiledbsoma SOMACollection
 #' @importFrom tiledbsoma SOMATileDBContext
 #' @export
 #'
 #' @examples
 open_soma <- function(census_version = "latest", uri = NULL, tiledbsoma_ctx = NULL) {
```

Comment:
should the default be `stable`?  That is the Python default.

Reply:
Yes; will do this shortly in #482